packages/story-editor/src/masks/output.js

```diff
@@ -77,35 +89,66 @@ export default function WithMask({
     );
   }
 
+  const borderWidth = border?.left;
+  const showSingleBorder = Boolean(singleBorderMask(element) && borderWidth);
+  const borderColor = border?.color
```

Comment:
If there's no `borderColor`, should we still display the border at all?

Agreed with Morten that `singleBorderMask` isn't the most self-explanatory name here, and it would be better to rename it to something like `hasMultiBorderSupport` or `canHaveMultiBorder` or sth similar.

Reply:
Okay, gotcha